src/field_theory/normal.lean

```diff
@@ -31,17 +31,28 @@ variables (F K : Type*) [field F] [field K] [algebra F K]
 
 /-- Typeclass for normal field extension: `K` is a normal extension of `F` iff the minimal
 polynomial of every element `x` in `K` splits in `K`, i.e. every conjugate of `x` is in `K`. -/
-@[class] def normal : Prop :=
-∀ x : K, is_integral F x ∧ splits (algebra_map F K) (minpoly F x)
+class normal : Prop :=
+(out' : ∀ x : K, is_integral F x ∧ splits (algebra_map F K) (minpoly F x))
```

Comment:
Why not two fields here?

Reply:
Well, forall is at the top level so I didn't change it. I guess it can be two fields.